### PR TITLE
Cursor 3d

### PR DESCRIPTION
**Pull request by wuzhuobin**
An exactly the same implementation of vtkCursor3D in c++. And this is the javascript version. 
I found out there is no cursor 3D in vtk js so i wrote one.

### CODE REVIEW

**Comment by wuzhuobin** [comment]
@jourdain  Thanks you for  your message. But i was too late to notice this script. lol

**Comment by jourdain** [comment]
Could you rename `Sources/Filters/Sources/vtkCursor3D/` to `Sources/Filters/Sources/Cursor3D`.
Also could you rework the history to have only one commit?
Then finally do you think you can add a test or example or both?

Thanks!

File: Sources/Filters/Sources/vtkCursor3D/index.js [hunk 1]
```diff
@@ -0,0 +1,475 @@
...
+    outData[0] = polyData;
+  };
+}
+
+// ----------------------------------------------------------------------------
+// Object factory
+// ----------------------------------------------------------------------------
+
+const DEFAULT_VALUES = {
+  focus: vtkPolyData.newInstance(),
```
[the remark this answers is not in the document]
Comment (wuzhuobin) [reply Sources/Filters/Sources/vtkCursor3D/index.js]: Thanks for your information.

**Comment by jourdain** [comment]
Yes please. Mark it WIP if you don't want it merge. ;-)

**Comment by jourdain** [comment]
An example or test or both would be nice.

**Pull request merged**